builtin/logical/pki/cert_util.go

```diff
@@ -1340,6 +1341,25 @@ func generateCreationBundle(b *backend, data *inputBundle, caSign *certutil.CAIn
 		}
 	}
 
+	// Parse SKID from the request for cross-signing.
+	var skid []byte
+	{
+		if rawSKIDValue, ok := data.apiData.GetOk("skid"); ok {
+			var err error
+			skidValue := rawSKIDValue.(string)
+
+			// Handle removing common separators to make copy/paste from tool
+			// output easier.
+			skidNoColonsSeparators := strings.ReplaceAll(skidValue, ":", "")
+			skidNoSpaceSeparators := strings.ReplaceAll(skidNoColonsSeparators, " ", "")
+			skidNoSeparators := strings.ReplaceAll(skidNoSpaceSeparators, "-", "")
```

Comment:
Would this be a little easier to extend/mess up by accidentally copy/pasting the wrong line and undoing an existing separator?

```go
skidNoSeparators := skidValue
for _, separator := range []string{":", "-", " "} {
    skidNoSeparators = strings.ReplaceAll(skidNoSeparators, separator, "")
}
```